src/get-started/quickstart.md

```diff
@@ -74,8 +88,9 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
I'm sorry, what do you mean by that?

Also, should we maybe put all of the runner specific instructions into tabs that are labeled with the runner? As it is now, we just have four consecutive boxes of shell commands that are not labeled. I imagine the confusion to only get bigger once we add more runners to this.